rpm/sound/sound.spec

```diff
@@ -40,7 +41,7 @@ install -m 644 %{_builddir}/qm-sound-%{version}/etc/containers/systemd/qm.contai
 %license LICENSE
 %doc README.md SECURITY.md
 %{_sysconfdir}/containers/systemd/qm.container.d/qm_dropin_mount_bind_snd.conf
-%{rootfs_qm}%{_sysconfdir}/containers/systemd/audio.container
+{qm_sysconfdir}/containers/systemd/audio.container
 
```

Comment:
it should be %{qm_sysconfdir}
 https://github.com/containers/qm/actions/runs/14243670624/job/39919214365

Reply:
Changed thank you